src/lib/analytics/apis.native.js

```diff
@@ -1,7 +1,19 @@
+// @flow
 import * as SentryNative from '@sentry/react-native'
 import amplitude from 'amplitude-js'
+import analytics from '@react-native-firebase/analytics'
+import { convertToGoogleAnalytics } from './utils'
+
+const googleWrapper = {
+  logEvent: (event, data: any = {}) => {
+    const eventData = convertToGoogleAnalytics(data)
+
+    analytics().logEvent(event, eventData)
+  },
+}
```

Comment:
analytics() should be called once. That because i supposed a singletone to be used:

```suggestion
class GoogleWrapper {
  constructor(analytics) {
    assign(this, { analytics })
  }
  
  logEvent(event, data: any = {}) {
    const eventData = convertToGoogleAnalytics(data)
    
    this.analytics.logEvent(event, eventData)
  }
}
```